### PR TITLE
Add the requests feature

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

**Comment by easeev** [comment]
@SivakovKonstantin @NikitosnikN please review

File: docs/platform/view-node-requests-metrics.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+To view the requests data:
+
+1. Click your project.
+1. Click your network.
+1. Click the node name.
+
+This will show you the data on the requests made to your node:
+
+* Requests made — a chart of the total requests over the selected period.
```
Comment (easeev) [inline docs/platform/view-node-requests-metrics.md:21]: HTTP requests, WebSocket requests, which requests?

File: docs/platform/view-node-requests-metrics.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Method calls — a breakdown of the method calls to the node over the selected period. Only available for HTTP requests.
+* Response codes — a breakdown of the response status codes from the Chainstack server proxying to the node. Any method call that reaches the node successfully, even if the method is invalid, will result in HTTP response `200`. Only available for HTTP requests.
```
Comment (easeev) [inline docs/platform/view-node-requests-metrics.md:23]: "from the Chainstack server proxying to the node" sounds too complicated and unnecessarily technical to me

File: docs/platform/view-node-requests-metrics.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    content: blockchain node HTTP WSS code stats request
+---
+
+# View node requests metrics
+
+You can view the data on requests made to your node.
+
+To view the requests data:
+
+1. Click your project.
```
Comment (easeev) [inline docs/platform/view-node-requests-metrics.md:15]: Should we specify that this is available only to public chain projects/network/nodes?

File: docs/platform/view-node-requests-metrics.md (hunk 1 above)
Comment (akegaviar) [reply docs/platform/view-node-requests-metrics.md]: Fixed

File: docs/platform/view-node-requests-metrics.md (hunk 2 above)
Comment (akegaviar) [reply docs/platform/view-node-requests-metrics.md]: Fixed

File: docs/platform/view-node-requests-metrics.md (hunk 3 above)
Comment (akegaviar) [reply docs/platform/view-node-requests-metrics.md:15]: Done.

**Review by PekopT — APPROVED** [review]

**Review by NikitosnikN — DISMISSED** [review]

File: docs/platform/view-node-requests-metrics.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+1. Click your network.
+1. Click the node name.
+
+This will show you the data on the requests made to your node:
+
+* Requests made — a chart of the total requests over HTTP and WebSocket for the selected period.
+* Method calls — a breakdown of the method calls to the node over the selected period. Only available for HTTP requests.
+* Response codes — a breakdown of the response status codes from the Chainstack platform. Any method call that reaches the node successfully, even if the method is invalid, will result in HTTP response `200`. Only available for HTTP requests.
```
Comment (easeev) [inline docs/platform/view-node-requests-metrics.md:23]: >a breakdown of the response status codes from the Chainstack platform. Any method call that reaches the node successfully, even if the method is invalid, will result in HTTP response `200`.

Guys @SivakovKonstantin @NikitosnikN @akegaviar why do we need this explanation, what is the purpose of it?

Comment (akegaviar) [reply docs/platform/view-node-requests-metrics.md:23]: We have the codes on the node details page and it's not immediately clear what they are for. If we have the codes there, we need to explain what they mean in terms of customers interacting with the nodes managed by Chainstack.

Maybe we don't need the codes there? Then we won't need this explanation.

Comment (easeev) [reply docs/platform/view-node-requests-metrics.md:23]: Keeping codes is ok, I just don't understand why we need such a long explanation. These are response codes for requests made to the HTTPS endpoint — that's it

Also worth having something like https://docs.alchemy.com/alchemy/documentation/error-reference#http-status-codes in the docs/KB

Comment (akegaviar) [reply docs/platform/view-node-requests-metrics.md]: okay, changed

**Review by NikitosnikN — APPROVED** [review]

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**